types/react-color/lib/helpers/color.d.ts

```diff
@@ -0,0 +1,11 @@
+import { HSLColor, HSVColor, RGBColor, ColorState } from "../../index";
+
+declare module "react-color/lib/helpers/color" {
```

Comment:
You shouldn't need a `declare module` here. You're already in the file `react-color/lib/helpers/color.d.ts`, so the contents of that file *are* the declaration of the module.